src/test/java/me/zeroeightsix/fiber/api/builder/constraint/ConstraintsBuilderTest.java

```diff
@@ -1,24 +1,7 @@
 package me.zeroeightsix.fiber.api.builder.constraint;
 
-import me.zeroeightsix.fiber.api.builder.ConfigAggregateBuilder;
-import me.zeroeightsix.fiber.api.builder.ConfigTreeBuilder;
-import me.zeroeightsix.fiber.api.constraint.CompositeType;
-import me.zeroeightsix.fiber.api.constraint.Constraint;
-import me.zeroeightsix.fiber.api.exception.RuntimeFiberException;
-import me.zeroeightsix.fiber.api.tree.ConfigLeaf;
-import me.zeroeightsix.fiber.api.tree.ConfigTree;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.function.Predicate;
-
-import static org.junit.jupiter.api.Assertions.*;
-
 class ConstraintsBuilderTest {
+/* TODO convert to a ConfigTypeTest
```

Comment:
Having an entire class commented out in `master` would be a bit ugly, mind either removing the class or performing the conversion?

Reply:
Looks like I had forgotten about that, will do the conversion